spread.yaml

```diff
@@ -101,8 +107,40 @@ debug-each: |
     journalctl -u snapd
     dmesg | grep DENIED
 
+rename:
+    # Move content into a directory, so that deltas computed by repack benefit
+    # from the content looking similar to codeload.github.com.
+    - s,^,$DELTA_PREFIX,S
+
+repack: |
+    # For Linode, compute a delta based on a known git reference that can be
+    # obtained directly from GitHub. There's nothing special about that reference,
+    # other than it will often be in the local repository's history already.
+    # The more recent the reference, the smaller the delta.
+    if ! echo $SPREAD_BACKENDS | grep linode; then
```

Comment:
$SPREAD_BACKEND holds the currently running backend.
$SPREAD_BACKENDS holds all currently running backends for the execution.

The question this answers is "if Linode is one of the backends at all, we use deltas".